tests/test_plots.py

```diff
@@ -0,0 +1,78 @@
+import scipy.stats as sps
+import numpy as np
+import unittest
+
+from GOFevaluation.utils import equiprobable_histogram
+from GOFevaluation.utils import plot_equiprobable_histogram
+
+
+class TestPlotEqualProbable(unittest.TestCase):
```

Comment:
Very nice!!
For completeness, you could add one call with `plot_mode='num_counts'` and one call with `order=[1, 0]` (or replace that for one of the calls). This way we would basically have all lines covered! 😊